heron/executor/src/python/heron_executor.py

```diff
@@ -976,6 +979,14 @@ def get_commands_to_run(self):
     # During shutdown the watch might get triggered with the empty packing plan
     if len(self.packing_plan.container_plans) == 0:
       return {}
+    if self._get_instance_plans(self.packing_plan, self.shard) is None and self.shard != 0:
```

Comment:
what is this for?

Reply:
if there is no container meta for this shard, keep shell only for health-check